<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_adrg</class>
 <widget class="QWidget" name="w_adrg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1660</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ADRG</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filterAdrg"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="AdrgTable">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="Desc">
          <property name="text">
           <string>DRG细分组</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="DrgTable"/>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>ADRG主诊池</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filterAdrgDiag"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="AdrgDiagPool"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>ADRG次诊池</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filterOtherDiag"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="OtherDiagPool"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>ADRG主手池</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filterAdrgOper"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="AdrgOperPool"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>MDC诊断池</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filterMdcDiag"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="MdcDiagPool"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
